release_notes.md

```diff
@@ -1,89 +1,3 @@
 
-### <a name="jpa-32"></a> Jakarta Persistence 3.2
```

Comment:
more out of curiosity 🙈 where does this file get used? I thought it was for the GH release, but I see that the release scripts have a template for that

Reply:
See https://github.com/hibernate/hibernate-orm/blob/main/ci/release/Jenkinsfile#L263

Reply:
That would change the content for all projects.  I did not want to do that (yet).

If y'all feel that is better, by all means feel free.  I think projects have a different pattern for the migration-guide url though.